### PR TITLE
Extend operators on Money SVO

### PR DESCRIPTION
**Pull request by Corniel**
- [x] `++ operator`
- [x] `-- operator`
- [x] `+ operator`
- [x] `- operator`
- [x] `+ operator (unary)`
- [x] `- operator (unary)`
- [x] `* operator`(s)
- [x] `/ operator`(s)
- [x] `Round()`
- [x] `Sum()`
- [x] `Average()`
- [x] `Min()`
- [x] `Max()`
- [x] Documentation!

### CODE REVIEW

File: src/Qowaiv/DecimalRound.cs [hunk 1]
```diff
@@ -0,0 +1,206 @@
...
+    public static class DecimalRound
+    {
+        private const int ScaleMask = 0x00FF0000;
+        private const int SignMask = unchecked((int)0x80000000);
+
+        public static decimal Round(this decimal value) => value.Round(0);
+        public static decimal Round(this decimal value, int decimals) => value.Round(decimals, DecimalRounding.AwayFromZero);
+        public static decimal Round(this decimal value, int decimals, int multiplyOf) => value.Round(decimals, DecimalRounding.AwayFromZero, multiplyOf);
+        public static decimal Round(this decimal value, int decimals, DecimalRounding rounding) => value.Round(decimals, rounding, 1);
+        public static decimal Round(this decimal value, int decimals, DecimalRounding rounding, int multiplyOf)
```
Comment (Sjaaky) [inline src/Qowaiv/DecimalRound.cs]: multipleOf?

File: src/Qowaiv/DecimalRound.cs [hunk 2]
```diff
@@ -0,0 +1,206 @@
...
+            var withMultiply = multiplyOf != 1;
+
+            var bits = decimal.GetBits(withMultiply ? value / multiplyOf : value);
+
+            int scale = (bits[3] & ScaleMask) >> 16;
+            var scaleDifference = scale - decimals;
+
+            if (scaleDifference <= 0)
+            {
+                return value;
```
Comment (Sjaaky) [inline src/Qowaiv/DecimalRound.cs]: What if you have multiple of?

File: README.md [hunk 3]
```diff
@@ -178,6 +178,12 @@ iban.ToString("F");// NL20 INGB 0001 2345 67
 Represents the amount and the currency. Technically this is not SVO. However it
 acts identically as a SVO.
 
+``` C#
+Money money = 125.34 + Currency.EUR;
+var sum = (12 + Currency.EUR) + (15 + Currency.EUR); // Throws CurrencyMismatchException()
```
Comment (Sjaaky) [inline README.md]: throws an exception, but the currencies are the same?

File: src/Qowaiv/DecimalRound.cs (hunk 2 above)
Comment (Corniel) [reply src/Qowaiv/DecimalRound.cs]: It returns the value, and if `scaleDifference <= 0` than after the division, is still has no decimal digits that should not be there. However, I slightly changed this mechanism anyway.

File: README.md (hunk 3 above)
Comment (Corniel) [reply README.md]: Oops.

**Pull request merged**